README.md

```diff
@@ -12,21 +12,45 @@ Setting up a fully working Linux system with all needed packages and openHAB rec
 openHABian aims to provide a **self-configuring** Linux system setup specific to the needs of every openHAB user.
 The project provides two things:
 
-*   A set of scripts to set up openHAB on any Debian/Ubuntu based system
-*   Complete **SD-card images pre-configured with openHAB** and many other openHAB- and Hardware-specific preparations, for the *Raspberry Pi* and the *Pine A64* platforms.
+*   a set of scripts to set up openHAB on any Debian based system (Raspbian, Ubuntu)
+*   a complete **SD-card image pre-configured with openHAB** and many other openHAB- and Hardware-specific preparations for all *Raspberry Pi* models.
 
-Closely related to openHABian is the repository [openhab-linuxpkg](https://github.com/openhab/openhab-linuxpkg) which provides linux packages for openHAB. openHABian uses those packages to setup openHAB with some additional configurations.
+## Hardware and OS support
+As of openHABian version 1.5.1 of the image, all Raspberry Pi models are supported as hardware.
+Anything x86 based may work or not. Anything else ARM based such as ODroids, OrangePis and the like may work or not.
+NAS servers such as QNAP and Synology boxes will not work. Support for PINEA64 was also dropped in this current release.
+We strongly recommend to choose Raspberry Pi 2, 3 or 4 systems to have 1GB of RAM or more. RPi 1 and 0/0W only have a single CPU core and 512MB. This can be sufficient to run a smallish openHAB setup, but it will not be enough to run a full-blown system with many bindings and memory consuming openHABian features/components such as ZRAM, InfluxDB or Grafana.
```

Comment:
Should be "We strongly recommend that users choose ..."

Reply:
done